js/script.js

```diff
@@ -0,0 +1,31 @@
+var url = 'https://restcountries.eu/rest/v1/name/';
+var list = document.querySelector('.container__list');
+
+document.querySelector('.container__button').addEventListener('click', function() {
+  search();
+});
+
+var search = function() {
+  var input = document.querySelector('.container__input').value;
+  if (!input.length) { input = 'Poland'; }
+  fetch(url + input)
+  .then(function(response) {
+    return response.json();
+  })
+  .then(showResults);
+}
+
+var showResults = function(response) {
+  list.innerHTML = '';
+  Array.prototype.forEach.call(response, function(item) {
```

Comment:
response will be an array, so you doesn't have to call forEach method from Array prototype ;)